gulp-core-build-sass/src/SassTask.ts

```diff
@@ -76,7 +78,13 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
 
     if (this.taskConfig.useCSSModules) {
       this.logVerbose('Generating css modules.');
-      return this._processFiles(gulp, srcPattern, completeCallback, modulePostCssPlugins);
+      return this._processFiles(gulp, srcPattern, completeCallback, modulePostCssPlugins,
+        (file: gulpUtil.File) => {
+          if (!path.basename(file.path).match(/module\.css$/)) {
```

Comment:
Yes, we do treat module.scss and scss files differently... And this warning is only written if we have "shouldUseModuleCss" turned on, but then don't use module.scss. I can introduce a flag to turn this on or off, if you'd like.